## This notebook is to visualize the corona affected region from 21-01-2020 to 14-02-2020.  

In [5]:



import folium
import pandas as pd
import numpy as np
import re



import os,glob
path="/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/"


all csv files are stored in a coronafile for merging purpose

In [6]:
coronafile=glob.glob(os.path.join(path,"*.csv"))
coronafile

['/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/02-09-2020_1030.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/01-30-2020_2130.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/02-02-2020_0500.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/02-01-2020_1800.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/01-29-2020_1430.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/01-24-2020_1200.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/02-08-2020_2304.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/02-04-2020_0800.csv',
 '/home/deepak/Documents/corona/COVID-19-master/archived_data/archived_daily_case_updates/01-21-2020_220

# all files are merged into single dataframe

In [7]:
all_df=[]
for f in coronafile:
    df=pd.read_csv(f,sep=",")
    all_df.append(df)
result=pd.concat(all_df)    
result

/home/deepak/anacondanew2/lib/python3.6/site-packages/ipykernel_launcher.py:5: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.

To accept the future behavior, pass 'sort=False'.

To retain the current behavior and silence the warning, pass 'sort=True'.

  """


Confirmed  ConfnSusp  Country/Region  Deaths          Last Update Notes  \
0     27100.0        NaN  Mainland China   780.0  2020-02-09 11:13:28   NaN   
1      1131.0        NaN  Mainland China     1.0  2020-02-09 15:03:05   NaN   
2      1075.0        NaN  Mainland China     0.0  2020-02-09 12:03:04   NaN   
3      1033.0        NaN  Mainland China     6.0  2020-02-09 11:23:08   NaN   
4       838.0        NaN  Mainland China     1.0  2020-02-09 12:33:01   NaN   
5       779.0        NaN  Mainland China     1.0  2020-02-09 09:33:02   NaN   
6       740.0        NaN  Mainland China     0.0  2020-02-09 11:33:12   NaN   
7       468.0        NaN  Mainland China     0.0  2020-02-09 10:23:11   NaN   
8       450.0        NaN  Mainland China     2.0  2020-02-09 12:13:14   NaN   
9       444.0        NaN  Mainland China     1.0  2020-02-09 15:03:05   NaN   
10      386.0        NaN  Mainland China     1.0  2020-02-09 12:43:01   NaN   
11      326.0        NaN  Mainland China     2.0  2020-02-09 03:43:02   NaN   
12      307.0        NaN  Mainland China     6.0  2020-02-09 08:43:05   NaN   
13      293.0        NaN  Mainland China     1.0  2020-02-09 06:33:01   NaN   
14      250.0        NaN  Mainland China     0.0  2020-02-09 09:13:11   NaN   
15      208.0        NaN  Mainland China     0.0  2020-02-09 03:03:09   NaN   
16      206.0        NaN  Mainland China     2.0  2020-02-09 12:43:01   NaN   
17      195.0        NaN  Mainland China     1.0  2020-02-09 00:33:07   NaN   
18      141.0        NaN  Mainland China     0.0  2020-02-09 08:53:02   NaN   
19      131.0        NaN  Mainland China     3.0  2020-02-09 13:33:02   NaN   
20      115.0        NaN  Mainland China     0.0  2020-02-09 09:03:04   NaN   
21      107.0        NaN  Mainland China     0.0  2020-02-09 12:23:03   NaN   
22       99.0        NaN  Mainland China     1.0  2020-02-09 09:53:02   NaN   
23       90.0        NaN  Mainland China     1.0  2020-02-09 04:03:07   NaN   
24       79.0        NaN  Mainland China     1.0  2020-02-09 11:13:28   NaN   
25       78.0        NaN  Mainland China     1.0  2020-02-09 09:03:04   NaN   
26       64.0        NaN          Others     0.0  2020-02-09 06:13:11   NaN   
27       54.0        NaN  Mainland China     0.0  2020-02-09 02:23:07   NaN   
28       45.0        NaN  Mainland China     0.0  2020-02-09 02:03:11   NaN   
29       45.0        NaN  Mainland China     0.0  2020-02-09 01:23:08   NaN   
..        ...        ...             ...     ...                  ...   ...   
14       17.0       17.0  Mainland China     NaN      1/25/2020  12am   NaN   
15       15.0       15.0  Mainland China     NaN      1/25/2020  12am   NaN   
16       10.0       14.0  Mainland China     NaN      1/25/2020  12am   NaN   
17        9.0        9.0  Mainland China     1.0      1/25/2020  12am   NaN   
18        8.0        8.0  Mainland China     NaN      1/25/2020  12am   NaN   
19        8.0        8.0  Mainland China     1.0      1/25/2020  12am   NaN   
20        6.0        6.0  Mainland China     NaN      1/25/2020  12am   NaN   
21        5.0        5.0  Mainland China     NaN      1/25/2020  12am   NaN   
22        5.0       41.0       Hong Kong     NaN      1/25/2020  12am   NaN   
23        5.0        5.0  Mainland China     NaN      1/25/2020  12am   NaN   
24        4.0        4.0  Mainland China     NaN      1/25/2020  12am   NaN   
25        4.0        4.0  Mainland China     NaN      1/25/2020  12am   NaN   
26        4.0        4.0  Mainland China     NaN      1/25/2020  12am   NaN   
27        3.0        4.0  Mainland China     NaN      1/25/2020  12am   NaN   
28        3.0        3.0          Taiwan     NaN      1/25/2020  12am   NaN   
29        3.0        3.0  Mainland China     NaN      1/25/2020  12am   NaN   
30        2.0        2.0           Macau     NaN      1/25/2020  12am   NaN   
31        2.0        2.0  Mainland China     NaN      1/25/2020  12am   NaN   
32        NaN        1.0  Mainland China     NaN      

In [8]:
result.columns

Index(['Confirmed', 'ConfnSusp', 'Country/Region', 'Deaths', 'Last Update',
       'Notes', 'Province/State', 'Recovered', 'Suspected'],
      dtype='object')

In [9]:
result.drop(columns=["ConfnSusp","Notes"],inplace=True,axis=1)

In [54]:
result.head(5)

Confirmed  Country/Region  Deaths          Last Update Province/State  \
0    27100.0  Mainland China   780.0  2020-02-09 11:13:28          Hubei   
1     1131.0  Mainland China     1.0  2020-02-09 15:03:05      Guangdong   
2     1075.0  Mainland China     0.0  2020-02-09 12:03:04       Zhejiang   
3     1033.0  Mainland China     6.0  2020-02-09 11:23:08          Henan   
4      838.0  Mainland China     1.0  2020-02-09 12:33:01          Hunan   

   Recovered  Suspected  
0     1480.0        NaN  
1      141.0        NaN  
2      201.0        NaN  
3      153.0        NaN  
4      186.0        NaN

In [11]:
result.reset_index(inplace=True,drop=True)


Confirmed  Country/Region  Deaths          Last Update  \
0       27100.0  Mainland China   780.0  2020-02-09 11:13:28   
1        1131.0  Mainland China     1.0  2020-02-09 15:03:05   
2        1075.0  Mainland China     0.0  2020-02-09 12:03:04   
3        1033.0  Mainland China     6.0  2020-02-09 11:23:08   
4         838.0  Mainland China     1.0  2020-02-09 12:33:01   
5         779.0  Mainland China     1.0  2020-02-09 09:33:02   
6         740.0  Mainland China     0.0  2020-02-09 11:33:12   
7         468.0  Mainland China     0.0  2020-02-09 10:23:11   
8         450.0  Mainland China     2.0  2020-02-09 12:13:14   
9         444.0  Mainland China     1.0  2020-02-09 15:03:05   
10        386.0  Mainland China     1.0  2020-02-09 12:43:01   
11        326.0  Mainland China     2.0  2020-02-09 03:43:02   
12        307.0  Mainland China     6.0  2020-02-09 08:43:05   
13        293.0  Mainland China     1.0  2020-02-09 06:33:01   
14        250.0  Mainland China     0.0  2020-02-09 09:13:11   
15        208.0  Mainland China     0.0  2020-02-09 03:03:09   
16        206.0  Mainland China     2.0  2020-02-09 12:43:01   
17        195.0  Mainland China     1.0  2020-02-09 00:33:07   
18        141.0  Mainland China     0.0  2020-02-09 08:53:02   
19        131.0  Mainland China     3.0  2020-02-09 13:33:02   
20        115.0  Mainland China     0.0  2020-02-09 09:03:04   
21        107.0  Mainland China     0.0  2020-02-09 12:23:03   
22         99.0  Mainland China     1.0  2020-02-09 09:53:02   
23         90.0  Mainland China     1.0  2020-02-09 04:03:07   
24         79.0  Mainland China     1.0  2020-02-09 11:13:28   
25         78.0  Mainland China     1.0  2020-02-09 09:03:04   
26         64.0          Others     0.0  2020-02-09 06:13:11   
27         54.0  Mainland China     0.0  2020-02-09 02:23:07   
28         45.0  Mainland China     0.0  2020-02-09 02:03:11   
29         45.0  Mainland China     0.0  2020-02-09 01:23:08   
...         ...             ...     ...                  ...   
3009       17.0  Mainland China     NaN      1/25/2020  12am   
3010       15.0  Mainland China     NaN      1/25/2020  12am   
3011       10.0  Mainland China     NaN      1/25/2020  12am   
3012        9.0  Mainland China     1.0      1/25/2020  12am   
3013        8.0  Mainland China     NaN      1/25/2020  12am   
3014        8.0  Mainland China     1.0      1/25/2020  12am   
3015        6.0  Mainland China     NaN      1/25/2020  12am   
3016        5.0  Mainland China     NaN      1/25/2020  12am   
3017        5.0       Hong Kong     NaN      1/25/2020  12am   
3018        5.0  Mainland China     NaN      1/25/2020  12am   
3019        4.0  Mainland China     NaN      1/25/2020  12am   
3020        4.0  Mainland China     NaN      1/25/2020  12am   
3021        4.0  Mainland China     NaN      1/25/2020  12am   
3022        3.0  Mainland China     NaN      1/25/2020  12am   
3023        3.0          Taiwan     NaN      1/25/2020  12am   
3024        3.0  Mainland China     NaN      1/25/2020  12am   
3025        2.0           Macau     NaN      1/25/2020  12am   
3026        2.0  Mainland China     NaN      1/25/2020  12am   
3027        NaN  Mainland China     NaN      1/25/2020  12am   
3028        1.0              US     NaN      1/25/2020  12am   
3029        1.0              US     NaN      1/25/2020  12am   
3030        2.0           Japan     NaN      1/25/2020  12am   
3031        5.0        Thailand     NaN      1/25/2020  12am   
3032        2.0     South Korea     NaN      1/25/2020  12am   
3033        3.0      Singapore      NaN      1/25/2020  12am   
3034        2.0         Vietnam     NaN      1/25/2020  12am   
3035        3.0          France     NaN      1/25/2020  12am   
3036        1.0       Australia     NaN      1/25/2020  12am   
3037        1.0           Nepal     NaN      1/25/2020  12am   
3038        3.0       Malaysia      NaN      1/25/2020  12am   

                    Province/State  R

In [65]:
df1=result.rename(columns={"Last Update":"Date"})
df1.head(5)

Confirmed  Country/Region  Deaths                 Date Province/State  \
0    27100.0  Mainland China   780.0  2020-02-09 11:13:28          Hubei   
1     1131.0  Mainland China     1.0  2020-02-09 15:03:05      Guangdong   
2     1075.0  Mainland China     0.0  2020-02-09 12:03:04       Zhejiang   
3     1033.0  Mainland China     6.0  2020-02-09 11:23:08          Henan   
4      838.0  Mainland China     1.0  2020-02-09 12:33:01          Hunan   

   Recovered  Suspected  
0     1480.0        NaN  
1      141.0        NaN  
2      201.0        NaN  
3      153.0        NaN  
4      186.0        NaN

In [68]:
df1
df1[['Date','Time']] = df1.Date.str.split(" ",expand=True,)
df1.head()

Confirmed  Country/Region  Deaths        Date Province/State  Recovered  \
0    27100.0  Mainland China   780.0  2020-02-09          Hubei     1480.0   
1     1131.0  Mainland China     1.0  2020-02-09      Guangdong      141.0   
2     1075.0  Mainland China     0.0  2020-02-09       Zhejiang      201.0   
3     1033.0  Mainland China     6.0  2020-02-09          Henan      153.0   
4      838.0  Mainland China     1.0  2020-02-09          Hunan      186.0   

   Suspected      Time  
0        NaN  11:13:28  
1        NaN  15:03:05  
2        NaN  12:03:04  
3        NaN  11:23:08  
4        NaN  12:33:01

In [97]:
df1["Date"].head()

0    2020-02-09
1    2020-02-09
2    2020-02-09
3    2020-02-09
4    2020-02-09
Name: Date, dtype: object

## few regions are unknown but countires are known therefore we replace such regions with country names

In [70]:
df1["Province/State"].replace(np.nan,df1["Country/Region"],inplace=True)
df1.head()

Confirmed  Country/Region  Deaths        Date Province/State  Recovered  \
0    27100.0  Mainland China   780.0  2020-02-09          Hubei     1480.0   
1     1131.0  Mainland China     1.0  2020-02-09      Guangdong      141.0   
2     1075.0  Mainland China     0.0  2020-02-09       Zhejiang      201.0   
3     1033.0  Mainland China     6.0  2020-02-09          Henan      153.0   
4      838.0  Mainland China     1.0  2020-02-09          Hunan      186.0   

   Suspected      Time  
0        NaN  11:13:28  
1        NaN  15:03:05  
2        NaN  12:03:04  
3        NaN  11:23:08  
4        NaN  12:33:01

# below we are finding latitudes and longitudes of all regions using geopy library
steps:
    1)creating an empty list and store all regions 
    
    
    2)create lat and long list for storing latitudes and longitudes of every region
    
    3)in loop of every region it goes into function and find its coordintes,if exception of timouet occurs,we send it again till we get
    
    4)here "Daimond cruise ship" is a ship affected with corona but not a region hence we cant find its coordinates ,we replace it with numpy nan value
    

# caution: run this once and store value somewhere else,as it is very time consumming for 3000+rows

l=[]
for i in (df1["Province/State"]):
    l.append(i)
print(l,sep="\n")    

# latitude and longitude printed below

In [49]:
lat

[31.15172525,
 23.1357694,
 29.0000001,
 34.0000001,
 27.9995878,
 32.0,
 28.0,
 33.0000001,
 29.5585712,
 36.0000001,
 30.5000001,
 39.906217,
 48.0000047,
 31.2322758,
 26.5450001,
 36.0,
 39.0000001,
 24.0,
 25.0,
 19.2000001,
 37.0,
 40.9975197,
 27.0,
 39.1235635,
 38.0000001,
 42.9995032,
 nan,
 43.2443242,
 37.0000001,
 41.7574769,
 1.357107,
 14.8971921,
 22.2793278,
 36.5748441,
 36.638392,
 35.40709525,
 23.9739374,
 4.5693754,
 51.0834196,
 13.2904027,
 46.603354,
 22.1757605,
 24.0002488,
 -22.1646782,
 -31.8759835,
 -36.5986096,
 55.001251,
 22.3511148,
 42.6384261,
 12.7503486,
 54.7023545,
 -30.5343665,
 43.653963,
 64.6863136,
 41.8755616,
 36.6248089,
 37.2333253,
 50.6402809,
 13.5066394,
 42.9836747,
 63.2467777,
 31.894343149999997,
 28.1083929,
 39.3262345,
 7.5554942,
 59.6749712,
 42.3602534,
 34.0536909,
 43.074761,
 45.8413676,
 47.6038321,
 33.4255056,
 31.15172525,
 29.0000001,
 23.1357694,
 34.0000001,
 27.9995878,
 28.0,
 32.0,
 29.5585712,
 36.0000001,
 30

In [55]:
long

[112.87832224656043,
 113.1982688,
 119.9999999,
 113.9999999,
 112.009538,
 117.0,
 116.0,
 119.9999999,
 106.5492822,
 118.9999999,
 102.4999999,
 116.3912757,
 127.999992,
 121.4692071,
 117.842778,
 109.0,
 116.0,
 109.0,
 102.0,
 109.5999999,
 112.0,
 122.9955469,
 107.0,
 117.1980785,
 101.9999999,
 125.9816054,
 nan,
 114.3251664,
 105.9999999,
 87.16738423046897,
 103.8194992,
 100.83273,
 114.1628131,
 139.2394179,
 127.6961188,
 95.95211573241954,
 120.9820179,
 102.2656823,
 10.4234469,
 108.4265113,
 1.8883335,
 113.5514142,
 53.9994829,
 144.5844903,
 147.2869493,
 144.6780052,
 -125.002441,
 78.6677428,
 12.674297,
 122.7312101,
 -3.2765753,
 135.6301212,
 -79.387207,
 97.7453061,
 -87.6244212,
 -121.1177379,
 -121.6846349,
 4.6667145,
 104.869423,
 -81.2496068,
 25.9209164,
 87.07813712706509,
 84.0917139,
 -4.8380649,
 80.7137847,
 14.5208584,
 -71.0582912,
 -118.2427666,
 -89.3837613,
 -74.2168946,
 -122.3300624,
 -111.9400125,
 112.87832224656043,
 119.9999999,
 113.1

In [51]:
lt=lat[:]

In [52]:
longt=long[:]

In [53]:
lt[-1]

4.5693754

In [71]:
df1.head(5)

Confirmed  Country/Region  Deaths        Date Province/State  Recovered  \
0    27100.0  Mainland China   780.0  2020-02-09          Hubei     1480.0   
1     1131.0  Mainland China     1.0  2020-02-09      Guangdong      141.0   
2     1075.0  Mainland China     0.0  2020-02-09       Zhejiang      201.0   
3     1033.0  Mainland China     6.0  2020-02-09          Henan      153.0   
4      838.0  Mainland China     1.0  2020-02-09          Hunan      186.0   

   Suspected      Time  
0        NaN  11:13:28  
1        NaN  15:03:05  
2        NaN  12:03:04  
3        NaN  11:23:08  
4        NaN  12:33:01

# inserting latitude and longitude into dataframe

In [72]:
df1["Latitude"]=lat
df1["Longitude"]=long

In [73]:
df1.head(5)

Confirmed  Country/Region  Deaths        Date Province/State  Recovered  \
0    27100.0  Mainland China   780.0  2020-02-09          Hubei     1480.0   
1     1131.0  Mainland China     1.0  2020-02-09      Guangdong      141.0   
2     1075.0  Mainland China     0.0  2020-02-09       Zhejiang      201.0   
3     1033.0  Mainland China     6.0  2020-02-09          Henan      153.0   
4      838.0  Mainland China     1.0  2020-02-09          Hunan      186.0   

   Suspected      Time   Latitude   Longitude  
0        NaN  11:13:28  31.151725  112.878322  
1        NaN  15:03:05  23.135769  113.198269  
2        NaN  12:03:04  29.000000  120.000000  
3        NaN  11:23:08  34.000000  114.000000  
4        NaN  12:33:01  27.999588  112.009538

In [74]:
df1=df1.rename(columns={"Province/State":"region"})

In [75]:
df1.head(5)

Confirmed  Country/Region  Deaths        Date     region  Recovered  \
0    27100.0  Mainland China   780.0  2020-02-09      Hubei     1480.0   
1     1131.0  Mainland China     1.0  2020-02-09  Guangdong      141.0   
2     1075.0  Mainland China     0.0  2020-02-09   Zhejiang      201.0   
3     1033.0  Mainland China     6.0  2020-02-09      Henan      153.0   
4      838.0  Mainland China     1.0  2020-02-09      Hunan      186.0   

   Suspected      Time   Latitude   Longitude  
0        NaN  11:13:28  31.151725  112.878322  
1        NaN  15:03:05  23.135769  113.198269  
2        NaN  12:03:04  29.000000  120.000000  
3        NaN  11:23:08  34.000000  114.000000  
4        NaN  12:33:01  27.999588  112.009538

# As dates are not in date format here tried to bring it to single  format 

In [98]:
import datetime

In [99]:
df1=df1.replace({'Date': {'/': '-'}}, regex=True)

In [100]:
df1.tail()

Confirmed Country/Region  Deaths       Date     region  Recovered  \
3034        2.0        Vietnam     0.0  1-25-2020    Vietnam        0.0   
3035        3.0         France     0.0  1-25-2020     France        0.0   
3036        1.0      Australia     0.0  1-25-2020  Australia        0.0   
3037        1.0          Nepal     0.0  1-25-2020      Nepal        0.0   
3038        3.0      Malaysia      0.0  1-25-2020  Malaysia         0.0   

      Suspected  Time   Latitude   Longitude          Y     M     D  
3034        0.0  None  13.290403  108.426511  1/25/2020  None  None  
3035        0.0  None  46.603354    1.888334  1/25/2020  None  None  
3036        0.0  None -24.776109  134.755000  1/25/2020  None  None  
3037        0.0  None  28.108393   84.091714  1/25/2020  None  None  
3038        0.0  None   4.569375  102.265682  1/25/2020  None  None

In [101]:
df1['date_new']=pd.to_datetime(df1.Date).dt.strftime('%d-%m-%Y')


In [105]:
df1.tail()

Confirmed Country/Region  Deaths       Date     region  Recovered  \
3034        2.0        Vietnam     0.0  1-25-2020    Vietnam        0.0   
3035        3.0         France     0.0  1-25-2020     France        0.0   
3036        1.0      Australia     0.0  1-25-2020  Australia        0.0   
3037        1.0          Nepal     0.0  1-25-2020      Nepal        0.0   
3038        3.0      Malaysia      0.0  1-25-2020  Malaysia         0.0   

      Suspected  Time   Latitude   Longitude     Y   M   D    date_new  
3034        0.0  None  13.290403  108.426511  2020  01  25  25-01-2020  
3035        0.0  None  46.603354    1.888334  2020  01  25  25-01-2020  
3036        0.0  None -24.776109  134.755000  2020  01  25  25-01-2020  
3037        0.0  None  28.108393   84.091714  2020  01  25  25-01-2020  
3038        0.0  None   4.569375  102.265682  2020  01  25  25-01-2020

In [103]:
df1[['D','M','Y']] = df1.date_new.str.split("-",expand=True,)


In [104]:
df1.head(5)

Confirmed  Country/Region  Deaths        Date     region  Recovered  \
0    27100.0  Mainland China   780.0  2020-02-09      Hubei     1480.0   
1     1131.0  Mainland China     1.0  2020-02-09  Guangdong      141.0   
2     1075.0  Mainland China     0.0  2020-02-09   Zhejiang      201.0   
3     1033.0  Mainland China     6.0  2020-02-09      Henan      153.0   
4      838.0  Mainland China     1.0  2020-02-09      Hunan      186.0   

   Suspected      Time   Latitude   Longitude     Y   M   D    date_new  
0        0.0  11:13:28  31.151725  112.878322  2020  02  09  09-02-2020  
1        0.0  15:03:05  23.135769  113.198269  2020  02  09  09-02-2020  
2        0.0  12:03:04  29.000000  120.000000  2020  02  09  09-02-2020  
3        0.0  11:23:08  34.000000  114.000000  2020  02  09  09-02-2020  
4        0.0  12:33:01  27.999588  112.009538  2020  02  09  09-02-2020

In [79]:
df1.shape

(3039, 13)

## there are few regions where recovered,death,confirmed cases missing values,hence replacing np.nan with 0 and deleting it


RESULT:9 rows gets delete

In [80]:
df1[["Suspected","Recovered","Deaths","Confirmed"]] = df1[["Suspected","Recovered","Deaths","Confirmed"]].replace( np.NaN,0)

In [81]:
 df1.drop(df1[(df1['Suspected'] == 0.0) & (df1['Recovered'] == 0.0)&(df1['Deaths'] == 0.0)&(df1['Confirmed'] == 0.0)].index,inplace=True)

In [82]:
df1.shape

(3030, 13)

## u might remember we stored values to latitude and longitude as nan for few rows as it had no location
we are storing it to port of yokohoma location because that is where the ship was halted 

In [90]:
df1["Latitude"] = df1["Latitude"].replace(np.nan,35.4498)

In [91]:
df1["Longitude"]=df1["Longitude"].replace(np.nan,139.6649)

# finally we are mapping using plugins to map called coronamap2 ,here it clusters are spreaded as we zoom,

In [110]:
from folium import plugins

# let's start again with a clean copy of the map of San Francisco
coronamap2 = folium.Map( zoom_start = 12, tiles='Stamen Terrain')

# instantiate a mark cluster object for the incidents in the dataframe
incidents = plugins.MarkerCluster().add_to(coronamap2)

# loop through the dataframe and add each data point to the mark cluster
for lat,lng,susp,confirm,death,d,m,y  in zip(df1["Latitude"], df1["Longitude"],df1["Suspected"],df1["Confirmed"],df1["Deaths"],df1["D"],df1["M"],df1["Y"]):
    folium.Marker(
        location=[lat, lng],
        icon=None,
        popup=("Date:"+str(d)+"-"+str(m)+"-"+str(y)+'<br>'+"Suspected:"+str(susp)+'<br>'+"Confirmed:"+str(confirm)+'<br>'+"Death:"+str(death)),
    ).add_to(incidents)

# display map
coronamap2

# this is without clustering

In [111]:
# create map and display it
coronamap1 = folium.Map( zoom_start=12)

# loop through the 100 crimes and add each to the map
for lat,lng,susp,confirm,death,d,m,y  in zip(df1["Latitude"], df1["Longitude"],df1["Suspected"],df["Confirmed"],df["Deaths"],df1["D"],df1["M"],df1["Y"]):
    folium.features.CircleMarker(
        [lat, lng],
        radius=5, # define how big you want the circle markers to be
        color='yellow',
        fill=True,
        popup=("Date:"+str(d)+"-"+str(m)+"-"+str(y)+'<br>'+"Suspected:"+str(susp)+'<br>'+"Confirmed:"+str(confirm)+'<br>'+"Death:"+str(death)),
        fill_color='blue',
        fill_opacity=0.6
    ).add_to(coronamap1)

# show map
coronamap1

# the below are latitude and longitudes can be used without running geopy code

abcd=(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(36.0, 109.0)
(39.0000001, 116.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(37.0, 112.0)
(40.9975197, 122.9955469)
(27.0, 107.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
nan nan
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(22.2793278, 114.1628131)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(4.5693754, 102.2656823)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(54.7023545, -3.2765753)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(28.0, 116.0)
(32.0, 117.0)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(36.0, 109.0)
(48.0000047, 127.999992)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(27.0, 107.0)
(42.9995032, 125.9816054)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(22.1757605, 113.5514142)
(31.894343149999997, 87.07813712706509)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(48.9467562, 11.4038717)
(63.2467777, 25.9209164)
(24.0002488, 53.9994829)
(12.7503486, 122.7312101)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(24.0, 109.0)
(36.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(37.0, 112.0)
(19.2000001, 109.5999999)
(38.0000001, 101.9999999)
(39.1235635, 117.1980785)
(27.0, 107.0)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(46.603354, 1.8883335)
(13.2904027, 108.4265113)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(50.000678, -86.000977)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(37.2333253, -121.6846349)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(19.2000001, 109.5999999)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(27.0, 107.0)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(41.7574769, 87.16738423046897)
(42.9995032, 125.9816054)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(51.0834196, 10.4234469)
(22.1757605, 113.5514142)
(46.603354, 1.8883335)
(13.2904027, 108.4265113)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(50.000678, -86.000977)
(42.6384261, 12.674297)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(-30.5343665, 135.6301212)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(63.2467777, 25.9209164)
(22.3511148, 78.6677428)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(12.7503486, 122.7312101)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(37.2333253, -121.6846349)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(27.9995878, 112.009538)
(34.0000001, 113.9999999)
(32.0, 117.0)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(28.0, 116.0)
(30.5000001, 102.4999999)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(24.0, 109.0)
(36.0, 109.0)
(25.0, 102.0)
(39.0000001, 116.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(48.0000047, 127.999992)
(39.1235635, 117.1980785)
(37.0, 112.0)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(37.0000001, 105.9999999)
(22.2793278, 114.1628131)
(27.0, 107.0)
(42.9995032, 125.9816054)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(48.9467562, 11.4038717)
(63.2467777, 25.9209164)
(24.0002488, 53.9994829)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(39.906217, 116.3912757)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(24.0, 109.0)
(31.2322758, 121.4692071)
(28.0, 116.0)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(32.0, 117.0)
(26.5450001, 117.842778)
(34.0000001, 113.9999999)

(33.0000001, 119.9999999)
(19.2000001, 109.5999999)
(39.1235635, 117.1980785)
(25.0, 102.0)
(36.0, 109.0)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(27.0, 107.0)
(42.9995032, 125.9816054)
(23.9739374, 120.9820179)
(37.0000001, 105.9999999)
(22.2793278, 114.1628131)
(22.1757605, 113.5514142)
(39.0000001, 116.0)
(38.0000001, 101.9999999)
(41.7574769, 87.16738423046897)
(37.0, 112.0)
(43.2443242, 114.3251664)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(41.8755616, -87.6244212)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(36.0, 109.0)
(39.0000001, 116.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(37.0, 112.0)
(40.9975197, 122.9955469)
(27.0, 107.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.152472, 127.068255)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(22.2793278, 114.1628131)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(4.5693754, 102.2656823)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(54.7023545, -3.2765753)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(35.40709525, 95.95211573241954)
(36.638392, 127.6961188)
(51.0834196, 10.4234469)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(22.3511148, 78.6677428)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(32.0, 117.0)
(39.906217, 116.3912757)
(29.5585712, 106.5492822)
(23.1357694, 113.1982688)
(24.0, 109.0)
(27.0, 107.0)
(19.2000001, 109.5999999)
(48.0000047, 127.999992)
(34.0000001, 113.9999999)
(22.2793278, 114.1628131)
(31.15172525, 112.87832224656043)
(27.9995878, 112.009538)
(28.0, 116.0)
(42.9995032, 125.9816054)
(40.9975197, 122.9955469)
(37.0000001, 105.9999999)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(30.5000001, 102.4999999)
(23.9739374, 120.9820179)
(39.1235635, 117.1980785)
(25.0, 102.0)
(29.0000001, 119.9999999)
(36.5748441, 139.2394179)
(38.8948932, -77.0365529)
(36.638392, 127.6961188)
(14.8971921, 100.83273)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(35.40709525, 95.95211573241954)
(36.638392, 127.6961188)
(51.0834196, 10.4234469)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(22.3511148, 78.6677428)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(39.906217, 116.3912757)
(36.0000001, 118.9999999)
(28.0, 116.0)
(32.0, 117.0)
(30.5000001, 102.4999999)
(26.5450001, 117.842778)
(31.2322758, 121.4692071)
(33.0000001, 119.9999999)
(24.0, 109.0)
(36.0, 109.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(25.0, 102.0)
(39.1235635, 117.1980785)
(48.0000047, 127.999992)
(39.0000001, 116.0)
(38.0000001, 101.9999999)
(37.0, 112.0)
(43.2443242, 114.3251664)
(22.2793278, 114.1628131)
(27.0, 107.0)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(7.9897371, -5.5679458)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.0000001, 119.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
nan nan
(19.2000001, 109.5999999)

(25.0, 102.0)
(27.0, 107.0)
(37.0, 112.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(37.0000001, 105.9999999)
(22.2793278, 114.1628131)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(32.9637838, -116.7706277)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(32.0, 117.0)
(30.5000001, 102.4999999)
(26.5450001, 117.842778)
(31.2322758, 121.4692071)
(28.0, 116.0)
(33.0000001, 119.9999999)
(24.0, 109.0)
(36.0, 109.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(25.0, 102.0)
(39.1235635, 117.1980785)
(48.0000047, 127.999992)
(39.0000001, 116.0)
(38.0000001, 101.9999999)
(37.0, 112.0)
(43.2443242, 114.3251664)
(22.2793278, 114.1628131)
(27.0, 107.0)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(22.1757605, 113.5514142)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(50.000678, -86.000977)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(23.9739374, 120.9820179)
(24.0002488, 53.9994829)
(46.603354, 1.8883335)
(48.9467562, 11.4038717)
(63.2467777, 25.9209164)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(29.5585712, 106.5492822)
(28.0, 116.0)
(32.0, 117.0)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(30.5000001, 102.4999999)
(26.5450001, 117.842778)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(24.0, 109.0)
(36.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(39.0000001, 116.0)
(48.0000047, 127.999992)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(27.0, 107.0)
(42.9995032, 125.9816054)
(23.9739374, 120.9820179)
(22.2793278, 114.1628131)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(48.9467562, 11.4038717)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(32.0, 117.0)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(28.0, 116.0)
(33.0000001, 119.9999999)
(24.0, 109.0)
(26.5450001, 117.842778)
(40.9975197, 122.9955469)
(19.2000001, 109.5999999)
(36.0, 109.0)
(48.0000047, 127.999992)
(25.0, 102.0)
(39.0000001, 116.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(37.0, 112.0)
(43.2443242, 114.3251664)
(22.2793278, 114.1628131)
(27.0, 107.0)
(42.9995032, 125.9816054)
(22.1757605, 113.5514142)
(41.7574769, 87.16738423046897)
(37.0000001, 105.9999999)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-24.7761086, 134.755)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.0000001, 119.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
nan nan
(19.2000001, 109.5999999)
(25.0, 102.0)
(27.0, 107.0)
(37.0, 112.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(43.2443242, 114.3251664)
(22.2793278, 114.1628131)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(32.9637838, -116.7706277)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(40.9975197, 122.9955469)
(19.2000001, 109.5999999)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(36.5748441, 139.2394179)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(22.3511148, 78.6677428)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)

(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(28.0, 116.0)
(32.0, 117.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(36.5748441, 139.2394179)
(22.2793278, 114.1628131)
(35.40709525, 95.95211573241954)
(36.638392, 127.6961188)
(51.0834196, 10.4234469)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(22.3511148, 78.6677428)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(29.5585712, 106.5492822)
(32.0, 117.0)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(30.5000001, 102.4999999)
(26.5450001, 117.842778)
(28.0, 116.0)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(24.0, 109.0)
(36.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(48.0000047, 127.999992)
(39.0000001, 116.0)
(39.1235635, 117.1980785)
(37.0, 112.0)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(27.0, 107.0)
(42.9995032, 125.9816054)
(23.9739374, 120.9820179)
(22.2793278, 114.1628131)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(48.9467562, 11.4038717)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(39.906217, 116.3912757)
(32.0, 117.0)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(31.2322758, 121.4692071)
(24.0, 109.0)
(28.0, 116.0)
(26.5450001, 117.842778)
(33.0000001, 119.9999999)
(19.2000001, 109.5999999)
(36.0, 109.0)
(40.9975197, 122.9955469)
(25.0, 102.0)
(48.0000047, 127.999992)
(39.1235635, 117.1980785)
(39.0000001, 116.0)
(37.0, 112.0)
(22.2793278, 114.1628131)
(43.2443242, 114.3251664)
(38.0000001, 101.9999999)
(27.0, 107.0)
(22.1757605, 113.5514142)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-24.7761086, 134.755)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(27.9995878, 112.009538)
(34.0000001, 113.9999999)
(32.0, 117.0)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(28.0, 116.0)
(30.5000001, 102.4999999)
(33.0000001, 119.9999999)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(31.2322758, 121.4692071)
(24.0, 109.0)
(36.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(37.0000001, 105.9999999)
(27.0, 107.0)
(42.9995032, 125.9816054)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(48.9467562, 11.4038717)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(39.906217, 116.3912757)
(32.0, 117.0)
(31.2322758, 121.4692071)
(34.0000001, 113.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(24.0, 109.0)
(19.2000001, 109.5999999)
(28.0, 116.0)
(26.5450001, 117.842778)
(33.0000001, 119.9999999)
(40.9975197, 122.9955469)
(36.0, 109.0)
(25.0, 102.0)
(39.1235635, 117.1980785)
(48.0000047, 127.999992)
(39.0000001, 116.0)
(43.2443242, 114.3251664)
(37.0, 112.0)
(22.2793278, 114.1628131)
(27.0, 107.0)
(42.9995032, 125.9816054)
(38.0000001, 101.9999999)
(37.0000001, 105.9999999)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-24.7761086, 134.755)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.0000001, 119.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(39.0000001, 116.0)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
nan nan
(19.2000001, 109.5999999)
(25.0, 102.0)
(27.0, 107.0)
(37.0, 112.0)
(39.1235635, 117.1980785)
(40.9975197, 122.9955469)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(4.5693754, 102.2656823)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)

(32.9637838, -116.7706277)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(29.4246002, -98.4951405)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(19.2000001, 109.5999999)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(27.0, 107.0)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(42.9995032, 125.9816054)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(22.1757605, 113.5514142)
(31.894343149999997, 87.07813712706509)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(48.9467562, 11.4038717)
(63.2467777, 25.9209164)
(24.0002488, 53.9994829)
(12.7503486, 122.7312101)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(59.6749712, 14.5208584)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(54.7023545, -3.2765753)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(32.0, 117.0)
(39.906217, 116.3912757)
(30.5000001, 102.4999999)
(31.2322758, 121.4692071)
(36.0000001, 118.9999999)
(28.0, 116.0)
(24.0, 109.0)
(33.0000001, 119.9999999)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(36.0, 109.0)
(39.1235635, 117.1980785)
(39.0000001, 116.0)
(25.0, 102.0)
(37.0, 112.0)
(43.2443242, 114.3251664)
(38.0000001, 101.9999999)
(27.0, 107.0)
(22.2793278, 114.1628131)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-24.7761086, 134.755)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(28.0, 116.0)
(32.0, 117.0)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(33.0000001, 119.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(24.0, 109.0)
(36.0, 109.0)
(25.0, 102.0)
(39.0000001, 116.0)
(48.0000047, 127.999992)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(42.9995032, 125.9816054)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(22.1757605, 113.5514142)
(31.894343149999997, 87.07813712706509)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(1.357107, 103.8194992)
(36.638392, 127.6961188)
(4.5693754, 102.2656823)
(46.603354, 1.8883335)
(48.9467562, 11.4038717)
(-31.8759835, 147.2869493)
(24.0002488, 53.9994829)
(-36.5986096, 144.6780052)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(13.2904027, 108.4265113)
(50.000678, -86.000977)
(-22.1646782, 144.5844903)
(42.6384261, 12.674297)
(54.7023545, -3.2765753)
(64.6863136, 97.7453061)
(38.8948932, -77.0365529)
(34.395342, -111.7632755)
(28.1083929, 84.0917139)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(63.2467777, 25.9209164)
(12.7503486, 122.7312101)
(22.3511148, 78.6677428)
(59.6749712, 14.5208584)
(32.0, 117.0)
(39.906217, 116.3912757)
(29.5585712, 106.5492822)
(26.5450001, 117.842778)
(38.0000001, 101.9999999)
(23.1357694, 113.1982688)
(24.0, 109.0)
(27.0, 107.0)
(19.2000001, 109.5999999)
(39.0000001, 116.0)
(48.0000047, 127.999992)
(34.0000001, 113.9999999)
(22.2793278, 114.1628131)
(31.15172525, 112.87832224656043)
(27.9995878, 112.009538)
(43.2443242, 114.3251664)
(33.0000001, 119.9999999)
(28.0, 116.0)
(42.9995032, 125.9816054)
(40.9975197, 122.9955469)
(22.1757605, 113.5514142)
(37.0000001, 105.9999999)
(35.40709525, 95.95211573241954)
(36.0, 109.0)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(37.0, 112.0)
(30.5000001, 102.4999999)
(23.9739374, 120.9820179)
(39.1235635, 117.1980785)
(31.894343149999997, 87.07813712706509)
(38.8948932, -77.0365529)
(41.7574769, 87.16738423046897)
(25.0, 102.0)
(29.0000001, 119.9999999)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(12.7503486, 122.7312101)
(4.5693754, 102.2656823)
(13.2904027, 108.4265113)
(-24.7761086, 134.755)
(19.4326296, -99.1331785)
(-10.3333333, -53.2)
(2.8894434, -73.783892)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(28.0, 116.0)
(32.0, 117.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(48.0000047, 127.999992)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(37.0, 112.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(36.5748441, 139.2394179)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(4.5693754, 102.2656823)
(22.1757605, 113.5514142)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-22.1646782, 144.5844903)
(-36.5986096, 144.6780052)
(22.3511148, 78.6677428)
(-30.5343665, 135.6301212)
(55.001251, -125.002441)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(24.0, 109.0)
(36.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(48.0000047, 127.999992)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(38.0000001, 101.9999999)
(39.1235635, 117.1980785)
(27.0, 107.0)
(42.9995032, 125.9816054)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)

(14.8971921, 100.83273)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(46.603354, 1.8883335)
(13.2904027, 108.4265113)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(50.000678, -86.000977)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(37.2333253, -121.6846349)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(23.1357694, 113.1982688)
(29.5585712, 106.5492822)
(32.0, 117.0)
(27.9995878, 112.009538)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(28.0, 116.0)
(33.0000001, 119.9999999)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(24.0, 109.0)
(36.0, 109.0)
(19.2000001, 109.5999999)
(39.0000001, 116.0)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(25.0, 102.0)
(39.1235635, 117.1980785)
(37.0, 112.0)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(27.0, 107.0)
(22.2793278, 114.1628131)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(7.9897371, -5.5679458)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(48.9467562, 11.4038717)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(29.5585712, 106.5492822)
(27.9995878, 112.009538)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(32.0, 117.0)
(30.5000001, 102.4999999)
(26.5450001, 117.842778)
(31.2322758, 121.4692071)
(28.0, 116.0)
(33.0000001, 119.9999999)
(24.0, 109.0)
(36.0, 109.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(25.0, 102.0)
(39.1235635, 117.1980785)
(48.0000047, 127.999992)
(39.0000001, 116.0)
(38.0000001, 101.9999999)
(37.0, 112.0)
(43.2443242, 114.3251664)
(22.2793278, 114.1628131)
(27.0, 107.0)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(22.1757605, 113.5514142)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-24.7761086, 134.755)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(13.5066394, 104.869423)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.0000001, 119.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(39.0000001, 116.0)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
nan nan
(25.0, 102.0)
(19.2000001, 109.5999999)
(27.0, 107.0)
(37.0, 112.0)
(39.1235635, 117.1980785)
(40.9975197, 122.9955469)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.0000001, 105.9999999)
(1.357107, 103.8194992)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(22.2793278, 114.1628131)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(4.5693754, 102.2656823)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(32.9637838, -116.7706277)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(29.4246002, -98.4951405)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(37.0, 112.0)
(27.0, 107.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
nan nan
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(36.5748441, 139.2394179)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(54.7023545, -3.2765753)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(29.5585712, 106.5492822)
(28.0, 116.0)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(33.0000001, 119.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(24.0, 109.0)
(39.0000001, 116.0)
(36.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(27.0, 107.0)
(37.0000001, 105.9999999)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(35.40709525, 95.95211573241954)
(31.894343149999997, 87.07813712706509)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)

(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(48.9467562, 11.4038717)
(63.2467777, 25.9209164)
(24.0002488, 53.9994829)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
nan nan
(27.0, 107.0)
(37.0, 112.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(41.7574769, 87.16738423046897)
(37.0000001, 105.9999999)
(22.2793278, 114.1628131)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(36.5748441, 139.2394179)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(13.2904027, 108.4265113)
(51.0834196, 10.4234469)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(32.9637838, -116.7706277)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(37.0, 112.0)
(27.0, 107.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
nan nan
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(36.5748441, 139.2394179)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(54.7023545, -3.2765753)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.0000001, 119.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(24.0, 109.0)
(36.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
nan nan
(27.0, 107.0)
(37.0, 112.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(22.2793278, 114.1628131)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(36.5748441, 139.2394179)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(32.9637838, -116.7706277)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(27.0, 107.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.152472, 127.068255)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(22.2793278, 114.1628131)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(4.5693754, 102.2656823)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(54.7023545, -3.2765753)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(48.0000047, 127.999992)
(25.0, 102.0)
(39.0000001, 116.0)
(40.9975197, 122.9955469)
(19.2000001, 109.5999999)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(27.0, 107.0)
(43.2443242, 114.3251664)
(42.9995032, 125.9816054)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)

(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(46.603354, 1.8883335)
(13.2904027, 108.4265113)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(50.000678, -86.000977)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(29.0000001, 119.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(48.0000047, 127.999992)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
nan nan
(19.2000001, 109.5999999)
(25.0, 102.0)
(27.0, 107.0)
(37.0, 112.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(43.2443242, 114.3251664)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(36.638392, 127.6961188)
(4.5693754, 102.2656823)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(32.9637838, -116.7706277)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(33.0000001, 119.9999999)
(29.5585712, 106.5492822)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(39.0000001, 116.0)
(36.0, 109.0)
(24.0, 109.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
nan nan
(37.0, 112.0)
(27.0, 107.0)
(40.9975197, 122.9955469)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(36.5748441, 139.2394179)
(35.40709525, 95.95211573241954)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(22.1757605, 113.5514142)
(54.7023545, -3.2765753)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(39.3262345, -4.8380649)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(28.0, 116.0)
(32.0, 117.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(48.0000047, 127.999992)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(36.5748441, 139.2394179)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(22.2793278, 114.1628131)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(4.5693754, 102.2656823)
(22.1757605, 113.5514142)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-22.1646782, 144.5844903)
(-36.5986096, 144.6780052)
(22.3511148, 78.6677428)
(-30.5343665, 135.6301212)
(55.001251, -125.002441)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(27.9995878, 112.009538)
(32.0, 117.0)
(29.5585712, 106.5492822)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(24.0, 109.0)
(26.5450001, 117.842778)
(33.0000001, 119.9999999)
(34.0000001, 113.9999999)
(19.2000001, 109.5999999)
(39.1235635, 117.1980785)
(28.0, 116.0)
(36.0, 109.0)
(27.0, 107.0)
(40.9975197, 122.9955469)
(22.2793278, 114.1628131)
(48.0000047, 127.999992)
(22.1757605, 113.5514142)
(41.7574769, 87.16738423046897)
(38.0000001, 101.9999999)
(25.0, 102.0)
(23.9739374, 120.9820179)
(37.0, 112.0)
(42.9995032, 125.9816054)
(39.0000001, 116.0)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(38.8948932, -77.0365529)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(32.0, 117.0)
(39.906217, 116.3912757)
(29.5585712, 106.5492822)
(26.5450001, 117.842778)
(38.0000001, 101.9999999)
(23.1357694, 113.1982688)
(24.0, 109.0)
(27.0, 107.0)
(19.2000001, 109.5999999)
(39.0000001, 116.0)
(48.0000047, 127.999992)
(34.0000001, 113.9999999)
(22.2793278, 114.1628131)
(31.15172525, 112.87832224656043)
(27.9995878, 112.009538)
(43.2443242, 114.3251664)
(33.0000001, 119.9999999)
(28.0, 116.0)
(42.9995032, 125.9816054)
(40.9975197, 122.9955469)
(22.1757605, 113.5514142)
(37.0000001, 105.9999999)
(35.40709525, 95.95211573241954)
(36.0, 109.0)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(37.0, 112.0)
(30.5000001, 102.4999999)
(23.9739374, 120.9820179)
(39.1235635, 117.1980785)
(31.894343149999997, 87.07813712706509)
(38.8948932, -77.0365529)
(41.7574769, 87.16738423046897)
(25.0, 102.0)
(29.0000001, 119.9999999)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)

(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(31.2322758, 121.4692071)
(39.906217, 116.3912757)
(26.5450001, 117.842778)
(24.0, 109.0)
(36.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(37.0, 112.0)
(19.2000001, 109.5999999)
(38.0000001, 101.9999999)
(39.1235635, 117.1980785)
(27.0, 107.0)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(4.5693754, 102.2656823)
(46.603354, 1.8883335)
(13.2904027, 108.4265113)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(50.000678, -86.000977)
(-22.1646782, 144.5844903)
(-30.5343665, 135.6301212)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(37.2333253, -121.6846349)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(30.5000001, 102.4999999)
(36.0000001, 118.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(48.0000047, 127.999992)
(25.0, 102.0)
(40.9975197, 122.9955469)
(19.2000001, 109.5999999)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(27.0, 107.0)
(37.0000001, 105.9999999)
(43.2443242, 114.3251664)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(1.357107, 103.8194992)
(36.638392, 127.6961188)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)
(4.5693754, 102.2656823)
(22.1757605, 113.5514142)
(46.603354, 1.8883335)
(13.2904027, 108.4265113)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(24.0002488, 53.9994829)
(-22.1646782, 144.5844903)
(50.000678, -86.000977)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(-30.5343665, 135.6301212)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(63.2467777, 25.9209164)
(22.3511148, 78.6677428)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(37.2333253, -121.6846349)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(28.0, 116.0)
(32.0, 117.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(27.0, 107.0)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(14.8971921, 100.83273)
(36.5748441, 139.2394179)
(22.2793278, 114.1628131)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(51.0834196, 10.4234469)
(4.5693754, 102.2656823)
(23.9739374, 120.9820179)
(22.1757605, 113.5514142)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(22.3511148, 78.6677428)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(64.6863136, 97.7453061)
(54.7023545, -3.2765753)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(55.001251, -125.002441)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(29.0000001, 119.9999999)
(23.1357694, 113.1982688)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(29.5585712, 106.5492822)
(28.0, 116.0)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(33.0000001, 119.9999999)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(24.0, 109.0)
(25.0, 102.0)
(39.0000001, 116.0)
(36.0, 109.0)
(19.2000001, 109.5999999)
(48.0000047, 127.999992)
(40.9975197, 122.9955469)
(37.0, 112.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(42.9995032, 125.9816054)
(41.7574769, 87.16738423046897)
(27.0, 107.0)
(22.2793278, 114.1628131)
(23.9739374, 120.9820179)
(35.40709525, 95.95211573241954)
(22.1757605, 113.5514142)
(31.894343149999997, 87.07813712706509)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.7014631, -118.7559974)
(34.395342, -111.7632755)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)
(50.000678, -86.000977)
(55.001251, -125.002441)
(13.5066394, 104.869423)
(7.5554942, 80.7137847)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(-22.1646782, 144.5844903)
(48.9467562, 11.4038717)
(63.2467777, 25.9209164)
(24.0002488, 53.9994829)
(12.7503486, 122.7312101)
(22.3511148, 78.6677428)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(34.0000001, 113.9999999)
(27.9995878, 112.009538)
(32.0, 117.0)
(28.0, 116.0)
(29.5585712, 106.5492822)
(33.0000001, 119.9999999)
(36.0000001, 118.9999999)
(30.5000001, 102.4999999)
(39.906217, 116.3912757)
(48.0000047, 127.999992)
(31.2322758, 121.4692071)
(26.5450001, 117.842778)
(36.0, 109.0)
(24.0, 109.0)
(39.0000001, 116.0)
(25.0, 102.0)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(37.0, 112.0)
(27.0, 107.0)
(39.1235635, 117.1980785)
(38.0000001, 101.9999999)
(42.9995032, 125.9816054)
(37.152472, 127.068255)
(43.2443242, 114.3251664)
(37.0000001, 105.9999999)
(41.7574769, 87.16738423046897)
(1.357107, 103.8194992)
(22.2793278, 114.1628131)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(35.40709525, 95.95211573241954)
(23.9739374, 120.9820179)
(51.0834196, 10.4234469)
(4.5693754, 102.2656823)
(22.1757605, 113.5514142)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-22.1646782, 144.5844903)
(24.0002488, 53.9994829)
(-31.8759835, 147.2869493)
(-36.5986096, 144.6780052)
(55.001251, -125.002441)
(22.3511148, 78.6677428)
(42.6384261, 12.674297)
(12.7503486, 122.7312101)
(54.7023545, -3.2765753)
(-30.5343665, 135.6301212)
(43.653963, -79.387207)
(64.6863136, 97.7453061)
(41.8755616, -87.6244212)
(36.6248089, -121.1177379)
(37.2333253, -121.6846349)
(50.6402809, 4.6667145)
(13.5066394, 104.869423)
(42.9836747, -81.2496068)
(63.2467777, 25.9209164)
(31.894343149999997, 87.07813712706509)
(28.1083929, 84.0917139)
(39.3262345, -4.8380649)
(7.5554942, 80.7137847)
(59.6749712, 14.5208584)
(42.3602534, -71.0582912)
(34.0536909, -118.2427666)
(43.074761, -89.3837613)
(45.8413676, -74.2168946)
(47.6038321, -122.3300624)
(33.4255056, -111.9400125)
(31.15172525, 112.87832224656043)
(23.1357694, 113.1982688)
(29.0000001, 119.9999999)
(29.5585712, 106.5492822)

(27.9995878, 112.009538)
(32.0, 117.0)
(39.906217, 116.3912757)
(31.2322758, 121.4692071)
(34.0000001, 113.9999999)
(30.5000001, 102.4999999)
(24.0, 109.0)
(36.0000001, 118.9999999)
(28.0, 116.0)
(33.0000001, 119.9999999)
(19.2000001, 109.5999999)
(40.9975197, 122.9955469)
(26.5450001, 117.842778)
(48.0000047, 127.999992)
(39.1235635, 117.1980785)
(39.0000001, 116.0)
(37.0, 112.0)
(25.0, 102.0)
(22.2793278, 114.1628131)
(36.0, 109.0)
(27.0, 107.0)
(42.9995032, 125.9816054)
(38.0000001, 101.9999999)
(37.0000001, 105.9999999)
(23.9739374, 120.9820179)
(41.7574769, 87.16738423046897)
(22.1757605, 113.5514142)
(43.2443242, 114.3251664)
(35.40709525, 95.95211573241954)
(38.8948932, -77.0365529)
(40.0796606, -89.4337288)
(36.5748441, 139.2394179)
(14.8971921, 100.83273)
(36.638392, 127.6961188)
(1.357107, 103.8194992)
(13.2904027, 108.4265113)
(46.603354, 1.8883335)
(-24.7761086, 134.755)
(28.1083929, 84.0917139)
(4.5693754, 102.2656823)


In [108]:
df1.to_csv("finalcopy.csv")